src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,33 @@
+.form-field {
+  box-sizing: border-box;
+  display: block;
+  height: 48px;
+  width: 100%;
+  padding: 14px 16px;
+  color: #292929;
+  background-color: #f8f8fa;
+  border: 1px solid transparent;
+  border-radius: 40px;
+
+  &::placeholder {
+    font-size: 14px;
+    line-height: 20px;
+    color: #7e7e83;
+  }
```

Comment:
Add hover on placeholder
`&:hover::placeholder {
    color: #292929;
  }`
This also works for :focus.